.github/workflows/deployBlocker.yml

```diff
@@ -48,7 +50,7 @@ jobs:
               channel: '#deployer',
               attachments: [{
                 color: "#DB4545",
-                text: '💥 New [Expensify.cash Deploy Blocker](${{ env.DEPLOY_BLOCKER_URL }}) found!',
+                text: '💥 New E.cash Deploy Blocker: ${{ env.DEPLOY_BLOCKER_TITLE }}',
```

Comment:
Don't you want the issue title to also be a link to the issue?

Reply:
Maybe dumb question but ... how can you text-encode a link in Slack? I always just use `CMD + Shift + U` to make Slack links

Reply:
Ah, [this SO](https://stackoverflow.com/questions/36294962/slack-markdown-for-links-are-not-resolving) makes me think that it might be possible like this (only via the Slack API):

```
<http://someurl|like this>
```

Reply:
Looked [in the source code for the action-slack repo](https://github.com/8398a7/action-slack/blob/master/src/fields.ts#L189) to confirm that this markdown formatting _should_ work, so I updated this PR to use it.